cursorless.nvim/README.md

```diff
@@ -0,0 +1,96 @@
+<!-- vim-markdown-toc GFM -->
+
+- [cursorless.nvim](#cursorlessnvim)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+    - [Lazy installation](#lazy-installation)
+    - [Manual installation](#manual-installation)
+  - [Configuration](#configuration)
+    - [neovim configuration](#neovim-configuration)
+    - [Talon configuration](#talon-configuration)
+  - [Frequently asked questions](#frequently-asked-questions)
+    - [nvim does not support Lazy?](#nvim-does-not-support-lazy)
+    - [nvim does not find the `neovim` globally installed package?](#nvim-does-not-find-the-neovim-globally-installed-package)
+
+<!-- vim-markdown-toc -->
+
+# cursorless.nvim
+
+Neovim plugin to support Cursorless
+
+## Prerequisites
+
+- neovim: https://neovim.io/
+- Talon voice: https://talonvoice.com/
+- neovim-talon: https://github.com/hands-free-vim/neovim-talon
+- node: https://nodejs.org/en
+- npm (generally part of node installation)
+- neovim node package: https://github.com/neovim/node-client (globally installed with `npm`)
+- (Optional) talon.nvim: https://github.com/hands-free-vim/talon.nvim
+
+## Installation
+
+Ideally, you want to use a neovim plugin manager like [lazy.nvim](https://github.com/folke/lazy.nvim).
+
+### Lazy installation
+
+After the typical [lazy setup](https://github.com/folke/lazy.nvim?tab=readme-ov-file#-installation), you'll have to add the `talon.nvim` plugin to your `init.lua`. We also recommend the following other plugins for the best experience.
+
+```lua
+require('lazy').setup({
+  -- Git related plugins
+  'tpope/vim-fugitive',
+
+  -- Optional: talon.nvim for best experience with neovim-talon
+  'hands-free-vim/talon.nvim',
+
+  'hands-free-vim/cursorless.nvim',
+
+  -- dependencies useful for neovim-talon
+  'dhruvasagar/vim-zoom',
+  'gcmt/taboo.vim',
+  'vim-scripts/BufOnly.vim'
```

Comment:
as above, are these really relevant to cursorless? maybe they should be recommended in talon.nvim?